frontend/src/pages/ActivityReport/Pages/components/Objective.js

```diff
@@ -187,7 +182,8 @@ export default function Objective({
         data.append('file', file);
       });
 
-      return uploadObjectivesFile(data);
+      const response = await uploadObjectivesFile(data);
```

Comment:
Just return response here. Not sure if this is from another PR.

Reply:
I changed this so that it would be properly caught in the try/catch block and therefore testable

Reply:
👍